src/main/kotlin/org/rust/ide/annotator/RsExternalLinterUtils.kt

```diff
@@ -261,8 +261,9 @@ private data class RsExternalLinterFilteredMessage(
                     .map { "${it.level.capitalize()}: ${StringEscapeUtils.escapeHtml(it.message)}" }
                     .forEach { add(it) }
 
-                joinToString("<br>") { formatMessage(it) }
-                    .replace(URL_REGEX) { url -> "<a href='${url.value}'>${url.value}</a>" }
+                formatMessage(link) +
+                    joinToString(separator = "<br>", prefix = "<br>") { formatMessage(it) }
+                        .replace(URL_REGEX) { url -> "<a href='${url.value}'>${url.value}</a>" }
```

Comment:
If we definitely want to escape html links in messages (BTW do we have such examples in tests?), maybe it's worth to escape links in the main message as well?

Something like
```suggestion
                buildString {
                    add(formatMessage(StringEscapeUtils.escapeHtml(message.message)).escapeUrls())
                    if (code != null) {
                        add(" [$code]")
                    }
                    add("<br>")
                    add(joinToString(separator = "<br>") { formatMessage(it) }.escapeUrls())
                }
```